services/datamanager/builtin/builtin.go

```diff
@@ -502,6 +526,7 @@ func (svc *builtIn) Update(ctx context.Context, cfg *config.Config) error {
 		if err := svc.initOrUpdateSyncer(ctx, 0, cfg); err != nil {
 			return err
 		}
+		// so if toggled sync off, then I want to STOP existing collectors
```

Comment:
Not necessarily - these should be independent. You might still want to capture data locally, just stop syncing it. Collectors are only in charge of collecting the data